packaging/docker/README.md

```diff
@@ -60,29 +60,14 @@ If you don't want to use the apps.plugin functionality, you can remove the mount
 
 ### Docker container names resolution
 
-If you want to have your container names resolved by netdata, you need to do two things:
-1) Make netdata user be part of the group that owns the socket.
+If you want to have your container names resolved by netdata, make netdata user be part of the group that owns the socket.
    To achieve that just add environment variable `PGID=[GROUP NUMBER]` to the netdata container,
```

Comment:
This shouldn't be indented any more.

Reply:
Also, please capitalize Netdata: 

```suggestion
   To achieve that just add environment variable `PGID=[GROUP NUMBER]` to the Netdata container,
```

I know there's a lot of inconsistency in the docs already but every change helps!